src-ui/src/app/components/common/input/date/date.component.ts

```diff
@@ -30,7 +35,20 @@ export class DateComponent
 
   placeholder: string
 
-  // prevent chars other than numbers and separators
+  onPaste(event: ClipboardEvent) {
```

Comment:
This function name tells you nothing, doesn't it?

HTML/Angular is the interface towards the browser, we cannot decide how event names are named there, but at least the typescript function names should be more handy?

`parsePastedDate()` would be a better name here, since this is, what the function is going to do, right?

Reply:
It's an event handler so the 'on...' Is a common convention I think. Also, actually, we don't know the contents of the paste since the user could be trying to paste in anything